hazelcast-sql/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -0,0 +1,265 @@
+<#--
+// Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+
+/**
+* Parses CREATE MAPPING statement.
```

Comment:
What is the advantage of `CREATE MAPPING` over `CREATE EXTERNAL/FOREIGN TABLE`? Are there any products that use a similar notion? AFAIK most products use the `TABLE` notion. This includes, but is not limited to, Postgres, SQL Server, Oracle, KSQL, Flink.

Also, when we have JDBC, these objects will be exposed as "tables" anyway, because this is how JDBC API is designed. And if, for example, a UI tool is connected to Hazelcast cluster, it will display these objects as tables also.